kong.conf.default

```diff
@@ -452,17 +452,32 @@
                                  # To read the file again after modifying it,
                                  # Kong must be reloaded.
 
-#dns_order = LAST,SRV,A,CNAME    # the order in which to resolve different
+#dns_order = LAST,SRV,A,CNAME    # The order in which to resolve different
                                  # record types. The `LAST` type means the
                                  # type of the last successful lookup (for the
                                  # specified name). The format is a (case
                                  # insensitive) comma separated list.
 
+#dns_stale_ttl = 4.0             # Once a record expires, how much longer (in
+                                 # seconds) may the current stale record be 
+                                 # used while the refresh query is executed in
+                                 # the background.
+                                 # So stale data will be used from expiry of a
+                                 # record until either the refresh query
+                                 # completes, or the `dns_stale_ttl` number of
+                                 # seconds have passed.
+
 #dns_not_found_ttl = 30.0        # ttl in seconds for empty DNS responses and
                                  # "(3) name error" responses.
 
 #dns_error_ttl = 1.0             # ttl in seconds for error responses.
 
+#dns_no_sync = off               # If enabled, then upon a cache-miss every
+                                 # request will trigger its own dns query.
+                                 # When disabled multiple requests for the
+                                 # same name/type will be synchronised to a
+                                 # single query.
```

Comment:
What I am curious about is, why would I be instructed to change this default - as in, under which circumstances? Since it seems to be the default is the most sane value and I see little (no) value in the old behavior compared to this new one (except, as mentioned, reverting to stability if something goes wrong).